library/std/src/sys/unix/ext/fs.rs

```diff
@@ -884,3 +884,27 @@ impl DirBuilderExt for fs::DirBuilder {
         self
     }
 }
+
+/// Change the root directory of the current process to the specified path.
+///
+/// This typically requires privileges, such as root or a specific capability.
+///
+/// This does not change the current working directory; you should call
+/// [`std::env::set_current_dir`][`crate::env::set_current_dir`] afterwards.
```

Comment:
Adding `extern crate self as std;` in std's lib.rs would avoid having to do the link target explicitly here (and definitely elsewhere too).

Reply:
According to @jyn514, doing that currently produces an ICE.

Reply:
This is https://github.com/rust-lang/rust/issues/73445. I haven't tried it in a while, it may work now.

Reply:
Also, you can shorten a bit like so:

```suggestion
/// [`std::env::set_current_dir`](crate::env::set_current_dir) afterwards.
```

Reply:
If you go the `extern crate` route, you might want to do

```rust
#[cfg(doc)]
extern crate self as std;
```

to ensure it's only used for docs.

Reply:
@camelid that will break any links that are re-exported, cfg(doc) isn't set for dependencies.

Reply:
> I haven't tried it in a while, it may work now.

Just tried it, `extern crate self as core` works but not `extern crate self as std` (https://github.com/rust-lang/rust/pull/84755, https://github.com/rust-lang/rust/issues/73445#issuecomment-830141476).